server/authz/testData/test-policy-map.csv

```diff
@@ -0,0 +1,9 @@
+# Policies go here
```

Comment:
To be consistent with the test data folder in ./test/, use `testdata` as the folder name.